packages/viewer/src/types/flags.ts

```diff
@@ -28,7 +28,14 @@ type Flag =
    * are larger than the supported maximum image size (1080p). By default
    * this option is disabled, and images will be scaled up to match the host.
    */
-  | 'letterboxFrames';
+  | 'letterboxFrames'
+
+  /**
+   * Enables or disables the default rotation interaction being changed to
+   * rotate around the mouse down location. This requires the enabling of
+   * depth buffers through the viewer's `streamAttributes`.
+   */
+  | 'defaultToSpinCenter';
```

Comment:
I think this deserves to be a top-level prop on the viewer. AFAIK, this is an option that is enabled via a toolbar button in Creo, which seems more like a prop you want to set on the viewer vs resetting the whole config.